### PR TITLE
Seperate Auth Pull Request

### PR DESCRIPTION
**Pull request by nnenna352**

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -122,7 +122,7 @@ <h1>Favorite SDKs</h1>
         };
 
         // Initialize Firebase
-            firebase.initializeApp(firebaseConfig);
+        firebase.initializeApp(firebaseConfig);
```
Comment (shrutip90) [inline src/main/webapp/index.html:126]: I see only this change (indentation fix) in the entire PR. Wondering if the PR was not created correctly. Did you want to add anything else here?

Comment (nnenna352) [reply src/main/webapp/index.html:126]: I made all the changes requested in the Page Adjustment Pr. I'm not sure how else I was supposed to move the file to another pr

Comment (shrutip90) [reply src/main/webapp/index.html:126]: Did you delete those changes in PageAddjustments PR? I cant see the diff if you dont...

Comment (nnenna352) [reply src/main/webapp/index.html:126]: I deleted the index.html file on the PageAdjustments branch

**Pull request merged**